lib/injekt.rb

```diff
@@ -0,0 +1,58 @@
+class Array
+
+  def injekt *args
+
+    if args.first.is_a? Integer
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.